conftest.py

```diff
@@ -9,3 +10,36 @@
 def user(db):
     """Create a user"""
     return UserFactory.create()
+
+
+@pytest.fixture
+def staff_jwt_token(admin_user):
+    """Creates a JWT token for a staff user"""
+    jwt_payload_handler = api_settings.JWT_PAYLOAD_HANDLER
+    jwt_encode_handler = api_settings.JWT_ENCODE_HANDLER
+
+    payload = jwt_payload_handler(admin_user)
+    payload['roles'] = ['staff']
+    return jwt_encode_handler(payload)
+
+
+@pytest.fixture
+def jwt_token(user):
+    """Creates a JWT token for a regular user"""
+    jwt_payload_handler = api_settings.JWT_PAYLOAD_HANDLER
+    jwt_encode_handler = api_settings.JWT_ENCODE_HANDLER
+
+    payload = jwt_payload_handler(user)
+    return jwt_encode_handler(payload)
+
+
+@pytest.fixture
+def staff_jwt_header(staff_jwt_token):
```

Comment:
is this fixture method called when the method signature of a test has `staff_jwt_token` as one of it's positional arguments? just wondering because I don't see this being imported or called directly anywhere.